cidbservice/app.py

```diff
@@ -55,13 +61,28 @@ def db_refresh(project_name):
     return db_service.refresh(project_name)
 
 
+@app.route("/db/refresh/<project_name>/<version>", methods=["GET"])
+def db_refresh_version(project_name, version):
+    check_authentication(project_name)
+    check_version(version)
+    return db_service.refresh("{}_{}".format(project_name, version))
```

Comment:
In fact this will not work, as it will fail to found the right user based on project name
https://github.com/akretion/cidbservice/blob/1547261ffea8273fd61e6e92e544afa47d20ef7d/cidbservice/tools.py#L127